docs/source/rfcs/LongTermStability.rst

```diff
@@ -0,0 +1,399 @@
+    branch is frozen permanently.
+
+A few points are worth taking notice here:
+
+ * Multiple LTS releases can co-exist before EOL
+
+ * Bug discovery can happen at any time
+
+ * The bugfix occurs ASAP after its discovery
+
+ * The severity of the defect determines the timing of the release
+
+   - moderate and minor bugfixes to lts branches are held until the
+     next sprint release
+
+   - critical bugs are released asynchronously, without waiting for
+     the next sprint release
+
+
+Avocado-VT
+==========
+
+Avocado-VT is an Avocado plugin that allows "VT tests" to be run
+inside Avocado.  It's a third-party project maintained mostly by
+Engineers from Red Hat QE with assistance from the Avocado team
+and other community members.
+
+It's a general consensus that QE teams use Avocado-VT directly
+from git, usually following the master branch, which they
+control.
+
+There's no official maintenance or stability statement for
+Avocado-VT.  Even though the upstream community is quite
+friendly and open to both contributions and bug reports,
+Avocado-VT is made available without any promises for
+compatibility or supportability.
+
+When packaged and versioned, Avocado-VT rpms should be considered
+just snapshots, available in packaged form as a convenience to
+users outside of the Avocado-VT development community.  Again,
+they are made available without any promises of compatibility or
+stability.
+
+* Which Avocado version should be used by Avocado-VT?
+
+  This is up to the Avocado-VT community to decide, but the
+  current consensus is that to guarantee some stability in
+  production environments, Avocado-VT should stick to a specific
+  LTS release of Avocado. In other words, the Avocado team
+  recommends production users of Avocado-VT not to install Avocado
+  from its master branch or upgrade it from Sprint Releases.
+
+  Given each LTS release will be maintained for 18 months, it
+  should be reasonable to expect Avocado-VT to upgrade to a new
+  LTS release once a year or so. This process will be done with
+  support from the Avocado team to avoid disruptions, with proper
+  coordination via the avocado mailing lists.
+
+  In practice the Avocado development team will keep watching
+  Avocado-VT to detect and document incompatibilities, so when
+  the time comes to do an upgrade in production, it's expected
+  that it should happen smoothly.
+
+* Will it be possible to use the latest Avocado and Avocado-VT
+  together?
+
+  Users are welcome to *try* this combination.  The Avocado
+  development team itself will do it internally as a way to monitor
+  incompatibilities and regressions.
+
+  Whenever Avocado is released, a matching versioned snapshot of
+  Avocado-VT will be made.  Packages containing those Avocado-VT
+  snapshots, for convenience only, will be made available in the
+  regular Avocado repository.
+
+Changelog
+=========
+
+Changes from `Version 4`_:
```

Comment:
@apahim it rendered just fine for me, but, since the other ones have newlines, I've (force) pushed a change with a new line here.  Thanks for noticing it.